src/test/kotlin/io/github/smiley4/ktorswaggerui/examples/ResourceExample.kt

```diff
@@ -0,0 +1,119 @@
+package io.github.smiley4.ktorswaggerui.examples
+
+import io.ktor.server.application.Application
+import com.fasterxml.jackson.core.util.DefaultIndenter
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter
+import com.fasterxml.jackson.databind.SerializationFeature
+import io.github.smiley4.ktorswaggerui.SwaggerUI
+import io.github.smiley4.ktorswaggerui.dsl.*
+import io.ktor.http.HttpStatusCode
+import io.ktor.resources.*
+import io.ktor.serialization.jackson.jackson
+import io.ktor.server.application.call
+import io.ktor.server.application.install
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import io.ktor.server.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.server.response.respond
+import io.ktor.server.routing.*
+
+/**
+ * Example to showcase usage with the resources plugin
+ */
+fun main() {
+    embeddedServer(Netty, port = 8080, host = "localhost", module = Application::myModule).start(wait = true)
+}
+
+@Resource("articles")
+class ArticlesRoute() {}
```

Comment:
I think the braces here can just be removed to fix the detekt-issue

```suggestion
class ArticlesRoute
```